ci/devenv-bootstrap.sh

```diff
@@ -14,6 +14,7 @@ EOF
 
 : PATH: "$PATH"
 
+cd "$HOME"
```

Comment:
curious why this bit is needed 🤔

Reply:
This is needed because the test executes inside a test repository. This confounds the coderoot detection.  I can now override it with the -d argument I had added before but that makes the run during the integration test less real-world.